packages/sdks/src/functions/__snapshots__/register-component.test.ts.snap

```diff
@@ -42,6 +42,7 @@ exports[`Component Registration and Serialization > serializeFn handles differen
     "func3": "return (function func3(x) {
           return x - 1;
         }).apply(this, arguments)",
+    "func4": "return (e => !0 === e.get(\\"isABTest\\")).apply(this, arguments)",
```

Comment:
it was appending `function` here, causing an invalid js syntax
<img width="692" alt="Screenshot 2024-11-12 at 12 31 23 PM" src="https://github.com/user-attachments/assets/29c527d6-1dfb-4d7c-90cd-24967fccabe9">